### PR TITLE
Issue #311

### PR DESCRIPTION
**Pull request by kirjaamo**
More banners added. Attempt to fix a linking issue in PDF, where "migration" target is not resolved.

### CODE REVIEW

File: galeracluster/source/documentation/administration.rst [hunk 1]
```diff
@@ -170,7 +181,7 @@ In addition to node administration, Galera Cluster also provides interfaces for
 
    Related Articles
 
-   - :doc:`../training/tutorials/migrate`
+   - :doc:`Migrate <../training/tutorials/migrate>`
```
Comment (byte) [inline galeracluster/source/documentation/administration.rst:184]: will this render correctly?

**Review by byte — COMMENTED** [review]
besides the one comment, about render, i guess this is ok, re:ads

**Comment by kirjaamo** [comment]
Neither link structure works in PDF. I have not yet found a solution for that problem.

**Comment by byte** [comment]
> Neither link structure works in PDF. I have not yet found a solution for that problem.

can i ask why we are adding ads to pdf's? maybe we exclude the ad's in the pdf's, and just have them in the website

**Comment by byte** [comment]
> > Neither link structure works in PDF. I have not yet found a solution for that problem.
> 
> can i ask why we are adding ads to pdf's? maybe we exclude the ad's in the pdf's, and just have them in the website
> 
> I don't know, really. Sakari asked me to add them for marketing purposes, I suppose. Do you think they should be removed from the PDF version? I think they just add noise to the PDF.

for me, i think its noise in the pdf version
can you please ask @jasenZ1 why ?

**Pull request closed without merging**